src/bindings/python/src/pyopenvino/frontend/frontend.cpp

```diff
@@ -204,7 +204,12 @@ void regclass_frontend_FrontEnd(py::module m) {
     fem.def(
         "add_extension",
         [](FrontEnd& self, const py::object& extension_path) {
-            return self.add_extension(Common::utils::convert_path_to_string(extension_path));
+            std::string model_path = Common::utils::convert_path_to_string(extension_path);
+#if defined(OPENVINO_ENABLE_UNICODE_PATH_SUPPORT) && defined(_WIN32)
+            return self.add_extension(ov::util::string_to_wstring(model_path));
```

Comment:
Is this change necessary? 
Currently we convert `string` to `wstring` on windows during loading of shared object, which is used by `add_extension()` method:
https://github.com/openvinotoolkit/openvino/blob/528f85c8aed1f615ebe564922e648145304a9715/src/common/util/src/os/win/win_shared_object_loader.cpp#L78

So it is expected that `add_extension()` works with unicode on windows. Is that not the case?

Reply:
`add_extension()` works, only the test was missing